src/ripple/consensus/Consensus.h

```diff
@@ -874,7 +875,7 @@ Consensus<Adaptor>::getJson(bool full) const
     if (mode_.get() != ConsensusMode::wrongLedger)
     {
         ret["synched"] = true;
-        ret["ledger_seq"] = previousLedger_.seq() + 1;
+        ret["ledger_seq"] = static_cast<std::uint32_t>(previousLedger_.seq())+ 1;
```

Comment:
I'm going to leave as is for now since it is just for JSON/debug support. I intend (eventually) to revisit how that is done in the generic code.